chapters/capabilities.adoc

```diff
@@ -707,6 +720,9 @@ ifdef::VK_QNX_external_memory_screen_buffer[]
   * ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_SCREEN_BUFFER_BIT_QNX for images
     only
 endif::VK_QNX_external_memory_screen_buffer[]
+ifdef::VK_EXT_external_memory_metal[]
+  * ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLTEXTURE_BIT_EXT
```

Comment:
Elsewhere in the MR, a VU for VkMemoryAllocateInfo says that VkDedicatedAllocationInfo must be used if the handle is MTLTEXTURE or MTLHEAP. Should MTLHEAP also be listed here? Should MTLHEAP be dropped from the VU?

Reply:
I just removed MTLHEAP from the VU since that was an oversight on my end. MTLHeaps can create resources (buffers and textures) so there's no need to make them dedicated. MTLTextures, on the other hand, do not allow to create buffers that point to the same address as long as they were created as an stand alone MTLTexture with no MTLHeap (this is why the MTLTEXTURE bit exists).

Thank you for catching this!